test/classification/test_scorer.py

```diff
@@ -1,25 +1,49 @@
 import unittest
+from typing import Tuple
 
 import numpy as np
 
 from snorkel.classification.scorer import Scorer
 
 
 class ScorerTest(unittest.TestCase):
-    def test_scorer(self):
+    def _get_labels(self) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
         golds = np.array([1, 0, 1, 0, 1])
         preds = np.array([1, 0, 1, 1, 0])
         probs = np.array([0.8, 0.6, 0.9, 0.7, 0.4])
+        return golds, preds, probs
 
+    def test_scorer(self) -> None:
         def sum(golds, preds, probs):
             return np.sum(preds)
 
-        scorer = Scorer(metrics=["accuracy", "f1"], custom_metric_funcs={"sum": sum})
+        scorer = Scorer(metrics=["accuracy", "f1"], custom_metric_funcs=dict(sum=sum))
```

Comment:
Let's rename this to "pred_sum" or something that doesn't conflict with a builtin method.